src/eko/couplings.py

```diff
@@ -36,102 +35,177 @@ def strong_coupling_mod_ev(mod_ev):
     raise ValueError(f"Unknown evolution mode {mod_ev}")
 
 
-@nb.njit("f8(u1,f8,u1,f8,f8)", cache=True)
-def as_expanded(order, as_ref, nf, scale_from, scale_to):
+@nb.njit("f8(f8,f8,f8)", cache=True)
+def expanded_lo(ref, beta0, lmu):
+    den = 1.0 + beta0 * ref * lmu
+    return ref / den
+
+
+@nb.njit("f8(f8,f8,f8,f8)", cache=True)
+def expanded_nlo(ref, beta0, beta1, lmu):
+    den = 1.0 + beta0 * ref * lmu
+    b1 = beta1 / beta0
```

Comment:
Maybe I dindn't understand what you mean. Here beta0, beta1, ecc are passed as parameters beacuse I wanted to use the same function for both QCD and QED since they have the same form but the beta functions are different

Reply:
I guess he meant: take as argument `ref, beta0, b1, lmu` and compute the actual `b1` in the calling function - this way you can *reuse* `b1` instead of *recompute*